instantlaunches/main.go

```diff
@@ -230,7 +228,7 @@ func New(db *sqlx.DB, group *echo.Group, init *Init) *App {
 }
 
 // swagger:parameters defaultsByVersion
-type defaultsByVersionParams struct {
+type defaultsByVersionParams struct { // nolint // says this is dead/unused, but I don't know for sure
```

Comment:
Not sure what happened here, but it said it was unused.

Reply:
It probably got missed during some refactoring. I'd recommend removing it.